config/manager/tcs_issuer.yaml

```diff
@@ -45,6 +45,7 @@ spec:
         - --health-probe-bind-address=:8083
         - --user-pin=$USER_PIN
         - --so-pin=$SO_PIN
+        - --use-random-nonce=false # Enable this when using KMRA version >= 2.2
```

Comment:
So default is false? Should it rather be true, since the older KMRA versions will eventually be updated to 2.2?

Reply:
My intention was not to _force_ the existing setup to upgrade the KMRA v2,2. Shall i make it to `true` instead ?

Reply:
I made it now default to `true`.